src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1775,6 +1797,14 @@ private unsafe List<string> AddDatabase(ClusterOperationContext context, Blittab
                     using (var databaseRecordAsJson = UpdateDatabaseRecordIfNeeded(databaseExists, shouldSetClientConfigEtag, index, addDatabaseCommand, newDatabaseRecord, context))
                     {
                         UpdateValue(index, items, valueNameLowered, valueName, databaseRecordAsJson);
+
+                        AssertLicenseLimits(type, serverStore, addDatabaseCommand.Record, items, context);
+
+                        foreach (var command in _licenseLimitsCommandsForCreateDatabase)
+                        {
+                            AssertLicenseLimits(command, serverStore, addDatabaseCommand.Record, items, context);
+                        }
```

Comment:
`LicenseLimitException` is an _expected_ exception, so the tx will be committed... 
In the line above you already do `UpdateValue` and put the database, so I expect it to throw and _accept_ the command.

But maybe I'm wrong :-)